packages/vit-s/src/hooks.js

```diff
@@ -322,3 +322,58 @@ export function useGetObsInfo(obsType, obsLabelsTypes, obsLabelsData, obsSetsMem
     return null;
   }, [obsType, obsLabelsTypes, obsLabelsData, obsSetsMembership]);
 }
+
+
+export const useMappedGeneList = (geneObject, featureMap) => {
```

Comment:
It seems worthwhile to add JSDoc documentation here to describe the inputs/outputs while they're still fresh in our mind and make it easier to revisit this in the future:
* https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html
* https://jsdoc.app/

Some examples where it's in use elsewhere in the repo: 
* Concrete example: https://github.com/vitessce/vitessce/blob/main/packages/file-types/zarr/src/AnnDataSource.js
* Other cases: https://github.com/search?q=repo%3Avitessce%2Fvitessce%20%2F%2F%20%40ts-check&type=code